set.mm

```diff
@@ -381010,7 +381010,15 @@ We intentionally choose to represent congruence (without loss of generality)
   as it is more convenient.  It is always possible to define ` dist ` for any
   particular geometry to produce equal results when conguence is desired, and
   in many cases there is an obvious interpretation of "distance" between
-  two points that can be useful in other situations.  A similar representation
+  two points that can be useful in other situations. Encoding congruence as
+  a distance function makes it easier to use these theorems in cases where
```

Comment:
Encoding congruence as distance equality makes...
or
Encoding congruence as an equality of distances makes...

Reply:
I agree. I slightly prefer the second one, so I'll use that. New version coming soon :-).